de.zray.Facetracker.json

```diff
@@ -0,0 +1,62 @@
+{
+  "app-id": "de.zray.Facetracker",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "command": "facetracker-gui",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=all",
+    "--socket=wayland"
+  ],
+  "modules": [
+    {
+      "name": "facetracker-addional-files",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -D de.zray.Facetracker.desktop /app/share/applications/",
+
+        "install -D facetracker/data/icons/de.zray.Facetracker.svg /app/share/icons/hicolor/scalable/apps",
+
+        "install -Dp -m 644 de.zray.Facetracker.metainfo.xml /app/share/metainfo/de.zray.Facetracker.metainfo.xml"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/Z-Ray-Entertainment/Facetracker",
+          "tag": "24.7a3"
+        }
+      ]
+    },
```

Comment:
Why is this source used twice? Just add these commands in `post-install` below where the actual source is.

Reply:
probably because the comment about "installed by upstream build system" got ignored. I think the whole module needs to be removed.